policyengine_us/variables/gov/states/sc/tax/income/sc_state_tax_addback.py

```diff
@@ -0,0 +1,43 @@
+from policyengine_us.model_api import *
+
+
+class sc_state_tax_addback(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "South Carolina State Tax addback"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://dor.sc.gov/forms-site/Forms/SC1040_2022.pdf"
+    defined_for = StateCode.SC
+
+    def formula(tax_unit, period, parameters):
+        p_us = parameters(period).gov.irs.deductions
+        itm_deds = [deduction for deduction in p_us.itemized_deductions]
```

Comment:
```suggestion
                itm_deds = [
            deduction
            for deduction in p_us.itemized_deductions
            if deduction not in ["salt_deduction"]
        ]
```

Reply:
Hi Parvel, why should we exclude the salt_deduction? Since we want to get line 17 from [Form 1040 Schedule A](https://www.irs.gov/pub/irs-pdf/f1040sa.pdf), I think state and local tax is included (line 5a).

Reply:
We have to remove the salt since it would create a circular reference, confirmed